Source/View Model/MapViewControllerModel.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  MapViewControllerModel.swift
+//  KnightHacks
+//
+//  Created by Patrick Stoebenau on 11/10/19.
+//  Copyright © 2019 KnightHacks. All rights reserved.
+//
+
+import Foundation
+
+internal class MapViewControllerModel {
+    
+}
```

Comment:
Is this being used for anything?

Reply:
This is not necessary and thanks for pointing this out. I deleted it.